.travis.yml

```diff
@@ -33,6 +33,10 @@ before_script:
   - psql -c "ALTER ROLE admin SET default_transaction_isolation TO 'read committed'" -U postgres
   - psql -c "ALTER ROLE admin SET timezone TO 'UTC'" -U postgres
   - psql -c "ALTER USER admin CREATEDB" -U postgres
+  - python manage.py makemigrations
+  - python manage.py migrate sites
+  - python manage.py migrate auth
+  - python manage.py migrate
```

Comment:
There is no need for line 37 and 38 since line 39 will automatically migrate for both the Django apps `sites` and `apps`